packages/block-editor/src/components/block-breadcrumb/index.js

```diff
@@ -60,7 +65,11 @@ function BlockBreadcrumb( { rootLabelText } ) {
 					<Button
 						className="block-editor-block-breadcrumb__button"
 						variant="tertiary"
-						onClick={ clearSelectedBlock }
+						onClick={ () => {
+							clearSelectedBlock();
+							// Focus the editor iframe.
+							blockRef.current?.closest( 'body' )?.focus();
```

Comment:
@ellatrix Is there a better way to get the editor canvas ref?

Reply:
Would it be worth trying the `.editor-styles-wrapper` classname, since that gets attached to the `body` element?

Reply:
Not really. But maybe, when selection is cleared, focus should automatically be set to the writing flow element or something (by the writing flow component, or iframe component maybe)

Reply:
See also https://github.com/WordPress/gutenberg/pull/62196/files#r1630727302